frontend/src/components/PumpAction.js

```diff
@@ -17,14 +20,63 @@ export default function PumpAction(){
     }
 
     return(
-        <Styled>
-            <input type={"number"} onChange={event => setMinutes(event.target.value)} placeholder={"Minuten"}/>
-            <button onClick={handleClickRunPumpButton}>run pump</button>
-            <button onClick={handleClickStopPumpButton}>stop pump</button>
-        </Styled>
+        <StyledComponent>
+            <MinuteButtons>
+                <Button className="Button" variant="outlined" onClick={()=>{setMinutes(5)}}>5'</Button>
```

Comment:
```suggestion
               [5,10, 15, 20].map(minutes =>  <Button className="Button" variant="outlined" onClick{()=>setMinutes(minutes)}}>{minutes}'</Button>

```

Reply:
if you now save the array of duration in the db and then use this data instead of the hardcoded array, you could easily offer the user the functionality to choose individual durations